components/server-side-render/index.js

```diff
@@ -46,9 +47,10 @@ export class ServerSideRender extends Component {
 		if ( null !== this.state.response ) {
 			this.setState( { response: null } );
 		}
-		const { block, attributes = {} } = props;
+		const { block, attributes = null } = props;
```

Comment:
Is it possible to have a `null` attributes? I thought we always fallback to `{}`?

Reply:
> Is it possible to have a `null` attributes?

Nope.

Reply:
That was based on this: https://github.com/WordPress/gutenberg/pull/7339#discussion_r196159751 but I'm happy to go either way :)

Reply:
@danielbachhuber should I update this to use `{}` instead of `null`?

Reply:
> should I update this to use `{}` instead of `null`?

No. `null` is more precise.